nixos/modules/rename.nix

```diff
@@ -285,6 +285,13 @@ with lib;
           throw "services.redshift.longitude is set to null, you can remove this"
           else builtins.fromJSON value))
 
+    # Redis
+    (mkRemovedOptionModule [ "services" "redis" "user" ] "The redis module now is hardcoded to the redis user.")
+    (mkRemovedOptionModule [ "services" "redis" "dbpath" ] "The redis module now uses /var/lib/redis as data directory.")
+    (mkRemovedOptionModule [ "services" "redis" "dbFilename" ] "The redis module now uses /var/lib/redis/dump.rdb as database dump location.")
+    (mkRemovedOptionModule [ "services" "redis" "appendOnlyFilename" ] "This option was never used.")
+    (mkRemovedOptionModule [ "services" "redis" "pidFile" ] "This option was removed.")
+
```

Comment:
Personally, I am in favor of removing those options. I fear there might be push-back, though.

Reply:
I don't think people care about the pidFile option, and appendOnlyFilename wasn't used at all.

Some might care about user, dbpath and dbfilename, but only if they changed the defaults - we previously also used /var/lib/redis/dump.rdb by default, as the `redis` user.

So in that case, nix will complain during rebuild (don't just spin up an empty redis) and people will be able to move the `dump.rbd` file to the new location.